lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -185,14 +303,13 @@ instance Buildable State where
             (\(k,v) -> fmt (""+|k|+": "+|length v|+"") )
             (Map.toList m)
 
-applyBlock :: (BlockHeader, PoolId) -> State -> State
-applyBlock (newTip, poolId) (State _prevTip prevMap) =
-        State newTip (Map.alter alter poolId prevMap)
+applyBlock :: Block -> State -> State
+applyBlock (Block newTip mpoolId) s@(State _prevTip prevMap) =
```

Comment:
are we still using this :thinking: ?

Reply:
No. I could remove in a commit in this PR.